docs/how_to/retrieve-job.rst

```diff
@@ -0,0 +1,21 @@
+Retrieve jobs from a retired system
+===================================
+
+See a list of retired systems on `this page <../retired.html>`__.
+
+To retrieve jobs from a retired system, you can use code similar to this:
+
+
+.. code-block:: python
+  
+  from qiskit_ibm_provider import IBMProvider
+
+  provider = IBMProvider(instance="hub/group/project")
+
+  #If you want to retrieve a list of jobs
+  jobs = provider.backend.jobs(backend_name=<backend_name>)
+
+  #If you want to retrieve a specific job you have the id for 
+  job = provider.backend.retrieve_job(<job_id>)
+
+The `provider.backend.jobs()` method also has more filtering options. Learn more from the `IBMBackend API documentation. <https://qiskit.org/documentation/partners/qiskit_ibm_provider/stubs/qiskit_ibm_provider.IBMBackend.html#qiskit_ibm_provider.IBMBackend>`__
```

Comment:
This code example is for qiskit-ibm-provider. We are in qiskit-ibm-runtime 😛 . 

So I think we'll need
1. Updated code to use qiskit-ibm-runtime
2. Something similar for the qiskit-ibm-provider doc (can be part of [this issue](https://github.com/Qiskit/qiskit-ibm-provider/issues/492))
3. Some words saying that qiskit-ibm-runtime can only be used to retrieve Qiskit Runtime jobs. And you need to use qiskit-ibm-provider to retrieve jobs submitted using ibm[q]-provider. (since old jobs are likely non-runtime jobs)